src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -895,6 +895,14 @@ public static bool LessThanOrEqualAll<T>(Vector<T> left, Vector<T> right)
         public static bool LessThanOrEqualAny<T>(Vector<T> left, Vector<T> right)
             where T : struct => LessThanOrEqual(left, right).As<T, nuint>() != Vector<nuint>.Zero;
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static Vector<T> LoadUnsafe<T>(ref T source, nint elementOffset)
```

Comment:
nit: The approved API surface for `Vector128` takes `nuint`, would be convenient to match this so less churn later.

Reply:
Same with `StoreUnsafe` below